contracts/security/Makefile

```diff
@@ -0,0 +1,38 @@
+UNAME_M := $(shell uname -m)
+
+.PHONY: all
+all: fmt build test lint schema optimize
+
+.PHONY: fmt
+fmt:
+	@cargo fmt --all -- --check
+
+.PHONY: build
+build:
+	@cargo wasm
+
+.PHONY: test
+test:
+	@RUST_BACKTRACE=1 cargo unit-test
+
+.PHONY: lint
+lint:
+	@cargo clippy -- -D warnings
+
+.PHONY: schema
+schema:
+	@cargo schema
+
+.PHONY: optimize
+optimize:
+ifeq ($(UNAME_M),arm64)
+	@docker run --rm -v $(CURDIR):/code \
```

Comment:
To support podman environments, lets add `:z` to the volume argument. It'll set the correct access labels:
e.g. `$(CURDIR):/code:z`

Reply:
Added.